tox.ini

```diff
@@ -1,7 +1,6 @@
 [tox]
 envlist =
     {py27,py34,py35,py36}-pyang{16,17,171,172,173},
```

Comment:
Surprised you didn't drop py27 support here, although perhaps we still need it to support some field stuff like `python-netconf-utilities`, not sure.

Reply:
yeah I considered dropping it, but thought I'd play it safe for now... this is also a major version bump so we could say that as of 2.0 we're dropping python2 support, but it wasn't too bad to maintain compatibility so I left it